ament_cmake_python/cmake/ament_python_install_package.cmake

```diff
@@ -122,6 +125,11 @@ setup(
     DESTINATION "${PYTHON_INSTALL_DIR}/"
   )
 
+  install(
+    DIRECTORY "${build_dir}/scripts/"
+    DESTINATION "lib/${package_name}/"
```

Comment:
shouldn't those be installed in `bin`?
or wherever the `setup.cfg` `install-scripts` option specified this to be

Reply:
> or wherever the setup.cfg install-scripts option specified this to be

That'd be the case if it was `setuptools` doing the actual installation. It's not. We are not even calling `install`, but `install_scripts` (a subset, different command line options). We _could_ let it build the layout somewhere and then install it manually. Let me see.

> shouldn't those be installed in `bin`?

We rarely install in `bin`, so I went for a sensible default. It's not necessary if layout replication works.

Reply:
Nope, I cannot infer from the `setup.cfg` in a robust way.